app/graphql/mutations/payment_providers/adyen/create.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Mutations
+  module PaymentProviders
+    module Adyen
+      class Create < Base
+        graphql_name 'AddAdyenPaymentProvider'
+        description 'Add Adyen payment provider'
+
+        input_object_class Types::PaymentProviders::AdyenInput
+
+        type Types::PaymentProviders::Adyen
```

Comment:
maybe this can also be moved into parent class

Reply:
That is how I originally wanted to implement it @lovrocolic 
But the create and update mutations use different input/argument. Also when I move the type to their parent class it autogenerates the return type of the mutation:

```
          "kind": "OBJECT",
          "name": "AddAdyenPaymentProviderPayload",
          "description": "Autogenerated return type of AddAdyenPaymentProvider",

```